cfpb/cfgov-refresh
# Add debt collection link to megamenu

Add debt collection link to mega menu behind feature flag for Thursday's release of debt collection portal

## Additions

- Debt collection link in `_vars-mega-menu.html` behind `DEBT_COLLECTION_RELEASE` flag 

## Testing

- Check out locally
- Create `DEBT_COLLECTION_RELEASE` flag in admin
- Verify that debt collection link appears in mega menu, under `Consumer Tools` after `Prepaid Cards` 

## Screenshots
<img width="1082" alt="screen shot 2017-01-10 at 9 12 10 am" src="https://cloud.githubusercontent.com/assets/778171/21816400/e7132232-d714-11e6-9730-9dfad6639d5a.png">


## Notes

- Can the `Consumer Credit Trends` and `Prepaid Cards` links be moved out from behind flags in `_vars-mega-menu.html`?


## Checklist
